pkgs/data/themes/catppuccin-kde/default.nix

```diff
@@ -2,23 +2,54 @@
 , stdenvNoCC
 , fetchFromGitHub
 , fetchpatch
-, flavour ? [ "frappe" ]
-, accents ? [ "blue" ]
-, winDecStyles ? [ "modern" ]
+, writeShellApplication
+, lightly-plasma
+, gnutar
+, unzip
+, flavour ? "frappe"
+, accent ? "blue"
+, winDecStyle ? "modern"
+, withLightly ? true
```

Comment:
It's been decided upstream that Lightly wouldn't be bundled by default with Catppuccin KDE. I think it is still valuable to keep this option, but set to false by default and maybe introduced in another PR